### PR TITLE
Model update

### PR DESCRIPTION
**Pull request by lyndond**
Changes:

- Fixed broken model and eigendistortion tests
- Changed model API
- returning torch.abs(eigenvals) because fisher matrix is PSD and numerical precision was giving neg eigenvals close to zero
- normalize power iterates by column norms intead of frobenius norm
- model default pad is now reflect and not circular

### CODE REVIEW

File: plenoptic/simulate/canonical_computations/filters.py [hunk 1]
```diff
@@ -52,7 +53,11 @@ def circular_gaussian2d(
         Filter kernel size. Recommended to be odd so that kernel is properly centered.
     std:
         Standard deviation of 2D circular Gaussian.
+<<<<<<< HEAD
```
Comment (pehf) [inline plenoptic/simulate/canonical_computations/filters.py:56]: 🧐

File: plenoptic/synthesize/eigendistortion.py [hunk 2]
```diff
@@ -327,11 +329,11 @@ def _synthesize_power(self,
 
         # note: v is an n x k matrix where k is number of eigendists to be synthesized!
         v = torch.randn(len(x), k).to(x.device)
-        v = v / v.norm()
+        v = v / torch.norm(v, dim=0, keepdim=True)
```
Comment (pehf) [inline plenoptic/synthesize/eigendistortion.py:332]: nice catch!

File: tests/test_metamers.py [hunk 3]
```diff
@@ -116,7 +116,11 @@ def test_coarse_to_fine(self, einstein_img, model, coarse_to_fine, tmp_path):
     @pytest.mark.parametrize('model', ['frontend.LinearNonlinear'], indirect=True)
     @pytest.mark.parametrize("clamp_each_iter", [True, False])
     def test_metamer_clamper(self, einstein_img, model, clamp_each_iter):
+<<<<<<< HEAD
```
Comment (pehf) [inline tests/test_metamers.py:119]: 🧐

**Comment by pehf** [comment]
This looks good! For CI to run, it might be necessary to merge main

File: tests/test_metamers.py (hunk 3 above)
Comment (lyndond) [reply tests/test_metamers.py:119]: nice catch

File: plenoptic/simulate/canonical_computations/filters.py (hunk 1 above)
Comment (lyndond) [reply plenoptic/simulate/canonical_computations/filters.py]: good catch

**Review by pehf — APPROVED** [review]

File: plenoptic/simulate/models/naive.py [hunk 4]
```diff
@@ -159,56 +184,87 @@ class CenterSurround(nn.Module):
...
 
         return filt
 
-    def _clamp_surround_std(self) -> Tensor:
+    def _clamp_surround_std(self):
         """Clamps surround standard deviation to ratio_limit times center_std"""
-        return self.surround_std.clamp(
-            min=self.width_ratio_limit * float(self.center_std), max=None
-        )
+        # TODO
```
Comment (nikparth) [inline plenoptic/simulate/models/naive.py:263]: confused about this? if it's just returning then why the rest of the code? that should be commented out as well if it's not being used or moved to another PR.

**Review by nikparth — CHANGES_REQUESTED** [review]
Everything looks good to me except those two small comments I made.

File: plenoptic/simulate/models/frontend.py [hunk 5]
```diff
@@ -362,43 +369,64 @@ def __init__(
...
+
+        self.contrast = Gaussian(
+           kernel_size=kernel_size,
+           out_channels=2,
+           pad_mode=pad_mode,
+           cache_filt=cache_filt,
         )
 
+        self.luminance_scalar = nn.Parameter(torch.rand(2) * 10)
+        self.contrast_scalar = nn.Parameter(torch.rand(2) * 10)
```
[the remark this answers is not in the document]
Comment (lyndond) [reply plenoptic/simulate/models/frontend.py:413]: Yes that's right. I'll comment.

File: plenoptic/simulate/models/naive.py (hunk 4 above)
Comment (lyndond) [reply plenoptic/simulate/models/naive.py:263]: Whoops nice catch. Fixed now and is no longer dead code.

**Review by nikparth — APPROVED** [review]
